app/Http/Controllers/API/v1/PageController.php

```diff
@@ -34,6 +34,40 @@ public function page_get_page(Domain $domain)
         return response($payload)->header('Content-Type', 'application/json');
     }
 
+    public function page_get_page_since_TIMESTAMP(Domain $domain, $timestamp)
+    {
+        $rule['timestamp'] = $timestamp;
+        Validator::make($rule, [
+            'timestamp' => 'required|integer|min:0',
+        ])->validate();
+
+        $pages = DB::table('pages')->select('id', 'slug', 'wd_page_id')->where('wiki_id', $domain->wiki_id)->where('metadata->wd_page_created_at','>',$timestamp)->whereNull('deleted_at')->orderBy('wd_page_id')->get();
```

Comment:
`orderBy('wd_page_id')`
Are other methods ordered by page ID?

Reply:
Every other method that references a timestamp is ordered by the chronological ID.

Reply:
Ah, fair enough, then.